profiles/dab_jwt/run_integration.sh

```diff
@@ -21,7 +21,9 @@ export HUB_ADMIN_PASS=admin
 export HUB_USE_MOVE_ENDPOINT=1
 export HUB_LOCAL=1
 export ENABLE_DAB_TESTS=1
-export HUB_TEST_MARKS="deployment_standalone"
+export HUB_TEST_MARKS="deployment_standalone and not skip_in_gw"
```

Comment:
"and not package and not iqe_ldap" I have also this in my tests, not sure if they should be excluded here thought

Reply:
hrm, lets check this in another PR if necessary. ATM all tests are passing, so the ldap stuff must be skipped.